lib/TBDGen/TBDGen.cpp

```diff
@@ -90,6 +90,11 @@ class TBDGenVisitor : public ASTVisitor<TBDGenVisitor> {
       return;
 
     auto declRef = SILDeclRef(VD);
+
+    // Transparent symbols don't exist, even if public.
+    if (declRef.isTransparent())
```

Comment:
There's an isSILOnly() method in IRGen somewhere. Eventually this will be 'isFragile()' instead, but for now let's abstract it away to not diverge.